documentation/developers.md

```diff
@@ -65,6 +65,42 @@ might be implemented to utilize the `compare-levels` function:
                                        :z "y"))}))
 ```
 
+### Implementing Alerts
+
+Alerts are returned by the  status function and allow you to expose human
+readable information about the state of your application. This makes it easy
+for status to be displayed and checked by unopinionated consumers. This works
+best if all conditions which may effect the availability of your service
+result in an alert.
+
+Examples:
+```clj
+{:severity :error
+ :message "Cannot connect to database"
+ :type :my-service/databas-unavailable
+ :details {}}
+
+{:severity :warning
+ :message "Repo may not be initialized"
+ :type :my-service/uninitilized-repo
+ :details { :repo "some-repo"} }
+```
+
+- `:severity` : The severity of the alert represents how consumers should support it.
```

Comment:
"function" and "respond" are not the same thing. A system that's not replicating will respond but should be marked as unhealthy for alerts and load balancing purposes.